tests/test_loaders_dumpers/input/book_series_lotr_yaml.yaml

```diff
@@ -0,0 +1,20 @@
+id: S001
```

Comment:
This is identical to the other file -- I would just have a single file, `book_series_lotr.yaml`.